src/Controller/CommandLineController.ts

```diff
@@ -0,0 +1,47 @@
+import yargs from 'yargs'
+
+export interface JunknetArguments {
+	makefile: string
+	docker_image: string
+	target: string | null
```

Comment:
Use `undefined` instead of `null` to indicate optional fields.

Reply:
I noticed that accessing nonexistent property names yielded `undefined`. 
I opted to use `null` in the union type to distinguish it from this case; most of my reasoning was based on [this](https://stackoverflow.com/questions/461966/why-is-there-a-null-value-in-javascript) post.  

Is using `undefined` in this way a JS convention?